Dependencies/UrlLib/Source/Unix/UrlRequest.cpp

```diff
@@ -174,7 +174,58 @@ namespace UrlLib
             if (curl)
             {
                 data.clear();
-                curl_easy_setopt(curl, CURLOPT_URL, m_url.data());
+
+                CURLUcode rc;
+                CURLU *url = curl_url();
+                rc = curl_url_set(url, CURLUPART_URL, m_url.c_str(), 0);
+                if (rc != CURLUE_OK)
+                {
+                    throw std::runtime_error("CURL: Unable to build URL.");
```

Comment:
nit again in a few places:
```suggestion
                    throw std::runtime_error{"CURL: Unable to build URL."};
```